region/norfair/east/Bat Cave.json

```diff
@@ -303,6 +328,30 @@
       "flashSuitChecked": true,
       "note": "Fall through the shot blocks and shoot around the Skree to open the door without falling into the lava to shinespark out of the room."
     },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharging, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canLongChainTemporaryBlue",
+        {"or": [
+          {"heatFrames": 550},
```

Comment:
Waiting for the bats to delete themselves isn't obvious since you can't see them.
Reaching the door is really awkward.  Bouncing helps, but really isn't safe.